### PR TITLE
Move the ellipsis and close icons to the icons library

### PR DESCRIPTION
**Pull request by youknowriad**
This PR moves two of the most used dashicons to the icons package:

 - ellipsis (kept as is)
 - no-alt (renamed close).

### CODE REVIEW

File: packages/block-editor/src/components/link-control/search-input.js [hunk 1]
```diff
@@ -85,7 +86,7 @@ const LinkControlSearchInput = ( {
 				disabled={ ! value.length }
 				type="reset"
 				label={ __( 'Reset' ) }
-				icon="no-alt"
+				icon={ close }
```
Comment (jasmussen) [inline packages/block-editor/src/components/link-control/search-input.js]: 💕

**Comment by jasmussen** [comment]
I recall back when we introduced the Ellipsis menu, I was frowned upon for calling it an "overflow menu". Which it technically is — menu items _overflow_ into that menu. 

But we ended up with some agreement that the menu could be called, the "More menu".

Incidentally, there's Material precedence for calling it that:

Screenshot 2020-01-30 at 10 22 19

But we'll need some direction too, as we will likely have both a horizontal more icon, and a vertical more icon. 

Whether we call the two icons `more-horiz` like Material, or `more-horizontal`, I have no strong opinion on... perhaps leaning ever so slightly to the latter. But up to you!

**Comment by youknowriad** [comment]
I like more-horizontal. Noting that today we rotate the icon in CSS and I believe we only use more-vertical in our UI. So should we change the SVG instead as well and name it more-vertical?

**Comment by jasmussen** [comment]
We will eventually want BOTH a vertical and a horizontal icon, and not use the SVG rotation (because we might want to use those for effects instead of foundational stuff). 

So probably call the icon `more-horizontal` for now, and keep the CSS rotation, and if/when we get a chance to do so, perhaps in the try/g3 branch or a separate g2 icons branch, we can add another one.

**Pull request merged**